heatpump.yaml

```diff
@@ -36,6 +40,8 @@ esphome:
   name: "${devicename}"
   comment: "${description}"
   friendly_name: "${description}"
+  project:
```

Comment:
I got an error on this one: `'version' is a required option for [project].` Sad, but let's remove it for now until I have good option to change the version automatically